articles/iot-hub/iot-hub-devguide-messages-read-custom.md

```diff
@@ -32,6 +32,10 @@ A single message may match the condition on multiple routing rules, in which cas
 An IoT hub has a default [built-in endpoint][lnk-built-in]. You can create custom endpoints to route messages to by linking other services in your subscription to the hub. IoT Hub currently supports Azure Storage containers, Event Hubs, Service Bus queues, and Service Bus topics as custom endpoints.
 
 > [!NOTE]
+> When you start using routing with custom endpoints, messages will no longer be delivered to the built-in endpoint unless they don't match any rules. In order to keep message delivery to the  built-in endpoint you can add a route and choose the endpoint named 'events'. 
```

Comment:
Can I suggest a slightly different wording?
When you use routing and custom endpoints, messages are only delivered to the built-in endpoint if they don't match any rules. To deliver messages to the built-in endpoint as well as to a custom endpoint, add a route that sends messages to the **events** endpoint.

Reply:
@katriendg - thanks again for your contribution. I've made a change directly to the content that should be published later today. Therefore I'm closing this PR.